commit 76545399b0e0e080c37f5af6cc1629fd3a2b7ffc
Author: Brian Bennett <brian.bennett@joyent.com>
Date:   2019-03-11T13:08:17-07:00 (7 months ago)
    
    joyent/node-manta#313 mget -H option documented incorrectly
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/CHANGES.md b/CHANGES.md
index 147eb47..65c1d75 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -4,6 +4,7 @@ See `CONTRIBUTING.md` for details on how to update this file
 
 ## not yet released
 
+- [#313](https://github.com/joyent/node-manta/issues/313) mget -H option documented incorrectly
 - [#358](https://github.com/joyent/node-manta/issues/358) mchmod command stopped working after v5.1.1
 - [#361](https://github.com/joyent/node-manta/issues/361) expires-relative short option has wrong case in man page
 - [#349](https://github.com/joyent/node-manta/issues/349) issue numbers in
diff --git a/docs/man/mget.md b/docs/man/mget.md
index 7187fe0..ef7b68f 100644
--- a/docs/man/mget.md
+++ b/docs/man/mget.md
@@ -1,4 +1,4 @@
-mget 1 "May 2013" Manta "Manta Commands"
+mget 1 "March 2019" Manta "Manta Commands"
 =======================================
 
 NAME
@@ -46,7 +46,7 @@ OPTIONS
   Authenticate as account (login name).
 
 `-H, --headers`
-  Print HTTP headers on stderr.
+  Additional HTTP headers to include.
 
 `-h, --help`
   Print a help message and exit.
diff --git a/man/man1/mget.1 b/man/man1/mget.1
index 74c2eec..7ec1ec2 100644
--- a/man/man1/mget.1
+++ b/man/man1/mget.1
@@ -1,4 +1,4 @@
-.TH mget 1 "May 2013" Manta "Manta Commands"
+.TH mget 1 "March 2019" Manta "Manta Commands"
 .SH NAME
 .PP
 mget \- download an object from Manta.
@@ -51,7 +51,7 @@ $ mfind \-t o \-n '.js$' ~~/stor/foo | xargs mget \-q | less
 Authenticate as account (login name).
 .TP
 \fB\fC\-H, \-\-headers\fR
-Print HTTP headers on stderr.
+Additional HTTP headers to include.
 .TP
 \fB\fC\-h, \-\-help\fR
 Print a help message and exit.
